js/web/lib/wasm/jsep/webgpu/ops/matmul.ts

```diff
@@ -134,8 +134,7 @@ export const createNaiveMatmulProgramInfo =
       return {
         name: 'MatMulNaive',
         shaderCache: {
-          hint: `${activationAttributes.activationCacheKey}_${components}_${aComponents}_${outputNumber}_${
```

Comment:
If you remove `activationCacheKey`, you need to make sure `clipMax`/`clipMin` is added as uniforms like other files. Or just add them to the hint. It makes me a little worried that it's too easy to miss some key words in hint and no case to catch it.

Reply:
clipMax/clipMin as uniforms here:
https://github.com/microsoft/onnxruntime/pull/18753/files#diff-a7970856887bffb786627723d75efe025f4e3ee2a8011926eeee8a9d6431aeb2R485

I agree that there is not enough cases to catch incorrect shader keys. And I am already start to thinking a better and general solution to catch this kind of issues.

Reply:
I forget to add clipMax/clipMin uniforms for the newly added MatmulNaive, updated.

For others, clipMax/clipMin as uniforms here:
https://github.com/microsoft/onnxruntime/pull/18753/files#diff-a7970856887bffb786627723d75efe025f4e3ee2a8011926eeee8a9d6431aeb2R485

I agree that there is not enough cases to catch incorrect shader keys. We need  a better and general solution to catch this kind of issues.